engine/db/table_segment_mvp.cpp

```diff
@@ -189,6 +195,11 @@ TableSegmentMVP::TableSegmentMVP(meta::TableSchema& table_schema, const std::str
 
     // Read the string table
     for (auto i = 0; i < record_number_; ++i) {
+      // skip deleted entry
+      if (deleted_->test(i)) {
```

Comment:
Can we skip? If we skip reading the file, the file cursor will messed up?

Reply:
good point. updated in 92dc14967714effffe4611e3103d35b9002d6a93